landing/index.jade

```diff
@@ -33,7 +33,7 @@ html.auth0-styleguide
         include templates/getting-started
         include templates/design
         include templates/components
-        //- include templates/react-components
+        // include templates/react-components
```

Comment:
//- is for invisible comments (will not show in the output HTML). "include templates/react-components" should appear? otherwise you can stay with "//" 👍 
reference: https://naltatis.github.io/jade-syntax-docs/#comments